### ISSUE TITLE
Enable Kestrel's libuv tests on the helix Windows arm64 queue

### ISSUE THREAD

**Issue by halter73** [dotnet/aspnetcore#22114]
The Windows arm64 helix agents are unable to find libuv.dll despite there being a Windows arm64 version of libuv.dll in the Libuv NuGet package.

```
System.DllNotFoundException : Unable to load DLL 'libuv' or one of its dependencies: The specified module could not be found. (0x8007007E)

at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.LibuvFunctions.NativeMethods.uv_loop_size()
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.LibuvFunctions.loop_size() in /_/src/Servers/Kestrel/Transport.Libuv/src/Internal/Networking/LibuvFunctions.cs:line 339
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.UvLoopHandle.Init(LibuvFunctions uv) in /_/src/Servers/Kestrel/Transport.Libuv/src/Internal/Networking/UvLoopHandle.cs:line 17
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.LibuvThread.ThreadStart(Object parameter) in /_/src/Servers/Kestrel/Transport.Libuv/src/Internal/LibuvThread.cs:line 288
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.LibuvConnectionListener.BindAsync() in /_/src/Servers/Kestrel/Transport.Libuv/src/Internal/LibuvConnectionListener.cs:line 141
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Tests.LibuvTransportTests.CallingAcceptAfterDisposeAsyncThrows() in /_/src/Servers/Kestrel/Transport.Libuv/test/LibuvTransportTests.cs:line 152
--- End of stack trace from previous location ---
```

Context: https://github.com/dotnet/aspnetcore/pull/22002#issuecomment-631998855

Once we discover why this is happening and fix the issue, we should enable Kestrel's libuv tests on the helix Windows arm64 queue.

**Comment by BrennanConroy** [comment]
libuv.dll currently goes into win-arm folder of the nuget pkg. Might need to be in win-arm64 folder or make a new dll for win-arm64.

**Comment by Pilchie** [comment]
Is the version in `win-arm` arm64 or arm32 though?

**Comment by Pilchie** [comment]
Based on the fact that there are both `linux-arm` and `linux-arm64`, I'm guessing that 32-bit is assumed and we need a new dll.

**Comment by halter73** [comment]
> Based on the fact that there are both linux-arm and linux-arm64, I'm guessing that 32-bit is assumed and we need a new dll.

There's no (w|l)in-arm32 RID either, so I'd say that's a very safe guess. We concluded the same thing in triage yesterday.

https://docs.microsoft.com/en-us/dotnet/core/rid-catalog

**Comment by JunTaoLuo** [comment]
This is a little trickier than initially thought and given that we deemed this as not a high priority item, I'm inclined to move this out. Specifically, we removed the configurations that builds libuv in teamcity since it was marked as obsolete and currently have no way of building it. We need to either restore the teamcity configurations, or better yet, just migrate the builds to azdo. Since it's not trivial, I'll move the work back to triage.
